example/src/App.js

```diff
@@ -0,0 +1,57 @@
+import logo from './logo.svg';
+import './App.css';
+import { trackSelfDescribingEvent, trackPageView, trackScreenView, trackStructEvent } from '@snowplow/webview-tracker';
+import { v4 as uuidv4 } from 'uuid';
+
+function trackEvent() {
```

Comment:
How about `trackMultipleEvents` or something along these lines, since we indeed track multiple events ?